File: 2219-maximum-number-of-words-found-in-sentences/2219-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max=0;
        for(int i=0;i<sentences.length;i++){
            int count=1,j=0;
            while(j<sentences[i].length()){
                if(sentences[i].charAt(j)==' '){
                    count++;
                }
                j++;
            }
            if(max<count){
                max=count;
            }
        }
        return max;
    }
}

File: 2219-maximum-number-of-words-found-in-sentences/2219-maximum-number-of-words-found-in-sentences.c
int mostWordsFound(char** sentences, int sentencesSize) {
    int max=0;
    for(int i=0;i<sentencesSize;i++){
        int j=0,count=1;
        while(sentences[i][j]){
            if(sentences[i][j] == ' '){
                count++;
            }
            j++;
        }
        if(max<count){
            max=count;
        }
    }
    return max;
}